README.md

```diff
@@ -28,7 +28,12 @@ sudo apt install build-essential cmake libxml2 libxml2-dev libcunit1 libcunit1-d
 
 ### Windows instructions
 
-[WINDOWS.md](WINDOWS.md)
+Download links for the latest Windows x86_64 binaries:
+
+* [MinGW](https://ci.appveyor.com/api/projects/mcuadros/libuast/artifacts/binaries.win64.mingw.zip?job=Environment:%20BUILD=mingw)
```

Comment:
Please upload the binaries to `Release` page, and not share a master compiled version.